README.md

```diff
@@ -452,7 +452,19 @@ Typed inject comes with only one implementation. The `rootInjector`. It implemen
 
 #### `injector.injectClass(injectable: InjectableClass)`
 
-This method creates a new instance of class `injectable` and returns it.
+This method creates a new instance of class `injectable` by populating its constructor arguments from the injector and returns it.
+
+Basically it is a shortcut for resolving values from the injector and creating a new instance with those values:
+
+```ts
+const args: any[] = this.resolveParametersToInject(Class, providedIn);
+return new Class(...(args as any));
```

Comment:
```suggestion
const logger = appInjector.resolve('logger');
const httpClient = appInjector.resolve('httpClient');
const service = new MyService(httpClient, logger);
```

I think this is a better example. I don't want to promote `args as any` stuff in the docs.